compiler/src/iree/compiler/Codegen/Common/TransformDialectStrategiesGPU.cpp

```diff
@@ -377,11 +386,12 @@ void SmallReductionStrategy::compute(int64_t maxNumThreadsToUse,
           mostMinorParallelDimensionSize, maxNumThreadsToUse);
 
   // Trailing elementwise unaligned tiling created bounded local buffers that
-  // are dynamic. Attempting to bound them in Common/PadDynamicAlloc.cpp results
-  // in a crash in the associated upstream util.
+  // are dynamic. Attempting to bound them in Common/PadDynamicAlloc.cpp
+  // results in a crash in the associated upstream util.
```

Comment:
This is not the first time I see this churn with comment reflow. Consider fixing the preferences of your editor to work with 80 cols....